openeo/api/logs.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+class LogEntry:
+    """
+    A log entry.
+    """
+
+    def __init__(self, entry):
+        self.id = entry['id']
+        self.code = entry.get("code", 0)
+        self.level = entry['level']
+        self.message = entry['message']
+        self.time =  entry.get("time", None) # todo: native date/time object?
```

Comment:
Is the "time" an official log field?

Reply:
Yes, since 1.1.0.

Actually, I also have to add the metrics here.

Reply:
Done, see also https://github.com/Open-EO/openeo-python-client/pull/206/files#r652520824